dandiapi/api/views/version.py

```diff
@@ -153,3 +153,28 @@ def publish(self, request, **kwargs):
 
         serializer = VersionSerializer(new_version)
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+    @swagger_auto_schema(
+        manual_parameters=[DANDISET_PK_PARAM, VERSION_PARAM],
+    )
+    def destroy(self, request, **kwargs):
+        """
+        Delete a version.
+
+        Deletes a version. Only published versions can be deleted, and only by
+        admin users.
```

Comment:
I believe this would not be visible in the swagger, and even "Delete a version" will not be ... use `operation_*` args to `swagger_auto_schema` to provide this information, as e.g. in https://github.com/dandi/dandi-api/pull/469/files#diff-fce128f68807723a60c1a0ebd67b9c11634a49450b51c5dc9ecac7b2273cdcbcR72

Reply:
I think if those parameters are omitted from `@swagger_auto_schema`, swagger will use the docstring instead.

Reply:
You are 100% correct on that. I mixed up with the "opposite issue" that single line docstrings do not show up as the summary (so need to be clicked on to see it)

![image](https://user-images.githubusercontent.com/39889/129637350-281dae48-43ee-459e-ad57-66fa5692d89c.png)

more of background on that: https://github.com/axnsan12/drf-yasg/issues/205#issuecomment-454331301 . So probably that should be addressed but unrelated to this PR.

Reply:
resolving by https://github.com/dandi/dandi-api/pull/482